### DISCUSSION TITLE
chia farm summary - error

### DISCUSSION THREAD

**Discussion by cnstrider** [Chia-Network/chia-blockchain#6974] [General]
hello,
   Could anybody tell me how to fix this ?   thanks for any information.
-------------------------------------------------------
 chia farm summary
Exception from 'wallet' {'error': "'NoneType' object has no attribute 'tx_store'
Farming status: Farming
Total chia farmed: Unknown
User transaction fees: Unknown
Block rewards: Unknown
Last height farmed: Unknown

**Comment by loppefaaret** [comment]
`chia wallet show`
